recipes/sparse-neighbors-search/meta.yaml

```diff
@@ -1,13 +1,14 @@
 package:
   name: sparse-neighbors-search
-  version: 0.2.3
+  version: 0.3
 
 source:
-  url: https://github.com/joachimwolff/minHashNearestNeighbors/archive/v0.2.3.tar.gz
-  md5: 49b3f9442b4caad82f8285c830296490
+  url: https://github.com/joachimwolff/minHashNearestNeighbors/archive/0.3.tar.gz
+  md5: bed04589d2c05c916a7b7ebdf31e71f2
+
 
 build:
-  skip: True  # [not py27]
+  skip: True  # [not py27 or win or osx]
```

Comment:
You should remove `win` here.  It also seems the osx build was working so is it any special reason why you are skipping it?